crates/context_aware_config/migrations/2024-05-06-133756_custom_types/up.sql

```diff
@@ -0,0 +1,40 @@
+-- Your SQL goes here
+CREATE TABLE jsonschema_types (
+    type_name TEXT PRIMARY KEY,
+    type_schema JSON NOT NULL,
+    created_by TEXT NOT NULL,
+    created_at TIMESTAMP NOT NULL,
+    last_modified TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL
+);
+CREATE INDEX IF NOT EXISTS json_schema_types_index ON ONLY public.jsonschema_types(type_name, created_at);
```

Comment:
Why to create an index over `created_at`?

Reply:
Answered in discussion

Reply:
https://github.com/juspay/superposition/discussions/52#discussioncomment-9516900

Reply:
https://github.com/juspay/superposition/discussions/52#discussioncomment-9523007